antora-playbook.yml

```diff
@@ -60,7 +60,7 @@ content:
     - url: https://github.com/couchbase/docs-server
       branches: [release/7.0, release/6.6, release/6.5, release/6.0]
     - url: https://github.com/couchbase/docs-sdk-common
-      branches: [release/6.6, release/6.5, release/6.0, release/5.5]
+      branches: [release/7.0, release/6.6, release/6.5, release/6.0, release/5.5]
```

Comment:
can we remove the 5.5  release?

Reply:
Yeah I guess should be fine to remove 5.5 from common. @RichardSmedley, I don't remember if Ray merged his PR for removing 5.x branches, but should be fine to go no?

Reply:
Yes, and anything else disappearing in the next 6 months or so - https://www.couchbase.com/support-policy/enterprise-software

Reply:
cf https://github.com/couchbase/docs-site/commit/cf8453f52b4f66beef23f6e32585a732661b62dd#diff-0e8a7c2f91f241f59cc684e048c14593122447d39c4b8bfa6953cb5e91151aa2

Reply:
btw https://github.com/couchbase/docs-site/pull/434 not yet merged, but no reason not to put saandbox ahead.  :+1: